examples/ice-phases/README.md

```diff
@@ -5,8 +5,16 @@ Nature Communicationsvolume 9, Article number: 2173 (2018)
 #
 * step 1
 
-gen_soap_descriptors.py -fxyz ice-dataset.xyz --rcut 4 --n 4 --l 6 --g 0.2 --periodic True --output xyz
+python ../../scripts/gen_soap_descriptors.py -fxyz ice-dataset.xyz --rcut 4 --n 4 --l 6 --g 0.2 --periodic True --output xyz
 
 * step 2
 
-pca.py -fxyz ASAP-n4-l6-c4.0-g0.2.xyz -fmat SOAP-n4-l6-c4.0-g0.2 -color ice-properties.dat --colorscolumn 3
+python ../../scripts/pca.py -fxyz ASAP-n4-l6-c4.0-g0.2.xyz -fmat SOAP-n4-l6-c4.0-g0.2 -color ice-properties.dat --colorscolumn 3
```

Comment:
why is there a need to use the relative path? The scripts are installed as well with everything else